insights/parsers/libvirtd_log.py

```diff
@@ -1,43 +1,70 @@
 """
+Libvirtd Logs
+=============
+
+This module contains the following parsers:
+
 LibVirtdLog - file ``/var/log/libvirt/libvirtd.log``
-====================================================
+----------------------------------------------------
+LibVirtdQemuLog - file ``/var/log/libvirt/qemu/*.log``
+------------------------------------------------------
+"""
+from insights.specs import Specs
+from .. import LogFileOutput, parser
```

Comment:
It's recommended to use a uniform format for the `import`ings in one single python module, either in all relative paths or in all full paths.
Since you have touched these lines in this parser,  would you please update them to a uniform format?

Reply:
Sure